types/async/test/index.ts

```diff
@@ -31,16 +31,6 @@ async.reject(['file1', 'file2', 'file3'], funcStringCbErrBoolean, (err: Error, r
 async.rejectSeries(['file1', 'file2', 'file3'], funcStringCbErrBoolean, (err: Error, results: string[]) => { });
 async.rejectLimit(['file1', 'file2', 'file3'], 2, funcStringCbErrBoolean, (err: Error, results: string[]) => { });
 
-async.parallel([
-    () => { },
-    () => { }
-], callback);
-
-async.series([
-    () => { },
-    () => { }
-]);
-
```

Comment:
These tests didn't cover anything that wasn't already covered by the other tests later on in this file.